doc/configuration.rst

```diff
@@ -817,6 +817,7 @@ Arguments:
     to pass before sending a newline to device.
   - console_ready (regex): optional, pattern used by the kernel to inform
     the user that a console can be activated by pressing enter.
+  - codec (str, default="utf-8"): optional, encoding used to decode output bytes
```

Comment:
Is "output" the bytes sent or received by labgrid? Shouldn't the encoding apply to both directions?